source/computer/commands.rst

```diff
@@ -44,6 +44,21 @@ Linux 下有成百上千个命令，每个命令都有众多选项。这一节
 cat
 ---
 
+``cat`` 命令名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来。
+
+将一个文件的内容打印到标准输出::
+
+    $ cat file
+
+将文件 :file:`file1` 和 :file:`file2` 合并到一个文件 :file:`target_file` 中::
+
+    $ cat file1 file2 > target_file
+
+打印文件内容并显示行号::
+
+    $ cat -n file
```

Comment:
I'm not sure if we should run these commands on some real files, so that users can try them.

Reply:
We can make an earthquake catalog with three eqs and a header.  https://github.com/seisman/SODrecipes/blob/master/eventArm/customEvents.csv.

Reply:
I hope we can find a few files which can be repeatedly used in all the commands.

Reply:
Yes. In our tomography tutorial, we use catalog.

I guess we can come back when we finish those commands related to files.

Reply:
Perhaps we can have all commands done in this PR, and then see what kind of files we need.

Reply:
Okay